src/main/kotlin/blackJack/domain/People.kt

```diff
@@ -0,0 +1,19 @@
+package blackJack.domain
+
+open class People(val name: String) {
```

Comment:
클래스 이름은 `Person`이 적절해 보여요.

```suggestion
open class Person(val name: String) {
```